course_info/tests/jasmine/controllers/PeopleControllerSpec.js

```diff
@@ -24,8 +56,9 @@ describe('Unit testing PeopleController', function() {
             };
         });
         scope = $rootScope.$new();
-        $routeParams.course_instance_id = 1234567890;
+        $routeParams.courseInstanceId = 1234567890;
```

Comment:
This is picky, but could this and the `courseInstanceURL` var defined above be derived from the same var somehow? Or is that more trouble/ambiguous than it's worth to make it DRYer?